include/pcmki/pcmki_output.h

```diff
@@ -21,6 +21,8 @@ extern pcmk__supported_format_t pcmk__out_formats[];
 int pcmk__out_prologue(pcmk__output_t **out, xmlNodePtr *xml);
 void pcmk__out_epilogue(pcmk__output_t *out, xmlNodePtr *xml, int retval);
 
+void pcmki__register_messages(pcmk__output_t *out);
+
```

Comment:
We're not using pcmki as a function prefix anywhere else, so I'm inclined to stick with pcmk__, but of course there's already a function with that name. But we can rename one or the other.

Reply:
Right, that's why I went with pcmki__ for this function.  I'd kind of like to change the name of the new function.  This is a situation where multiple naming schemes just happen to collide.  Internal functions get named pcmk__, but message registering functions get named _register_messages.